docker-compose.yaml

```diff
@@ -4,26 +4,26 @@ services:
       dockerfile: Dockerfile.frontend
     environment:
       - API_SPEC=./openapi.yaml
-      - VITE_API_URL=http://localhost:8080
-      - NODE_ENV=development
+      - API_URL=http://backend:8080
```

Comment:
mmmm ive seen the port number so many times as an env var is there a reason why not here?

Reply:
in other projects in my life i mean not in this pr or this repo